### PR TITLE
[WIP Feature/553 setting up behave

### PR DESCRIPTION
**Pull request by w1stler**
__Story / Bug id:__ https://github.com/CodeForPoznan/volontulo/issues/553

__Reference person:__ @w1stler

__Description:__

Setting up behave tests

### CODE REVIEW

File: behave/requirements.pip [hunk 1]
```diff
@@ -1,2 +1,2 @@
 selenium
-behave
+behave
```
Comment (sabinaczopik) [inline behave/requirements.pip]: newline at end of file

File: behave/sample_test.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
```
Comment (sabinaczopik) [inline behave/sample_test.py]: unused import

Comment (sabinaczopik) [inline behave/sample_test.py]: unused import

File: behave/sample_test.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+class VolontuloLogin(unittest.TestCase):
+
+    def setUp(self):
+        self.driver = webdriver.Chrome()
+
+    def test_volontulo_login(self):
+        driver = self.driver
+        driver.get("http://volontuloapp.org")
+        elem = driver.find_element_by_xpath(
+                "//div/a[contains(text(), 'Zaloguj się')]")
```
Comment (sabinaczopik) [inline behave/sample_test.py]: unused variable, you can use click() in one line with find_element._by_xpath

File: behave/sample_test.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        elem.click()
+        elem = driver.find_element_by_xpath(
+                "//div/h2[contains(text(), 'Logowanie')]"
+                "//..//form/div/input[contains(@id, 'id_email')]"
+            )
+        elem.send_keys("volontulo.personas+user@o2.pl")
+
```
Comment (sabinaczopik) [inline behave/sample_test.py]: the same as above

File: behave/sample_test.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        elem = driver.find_element_by_xpath(
+                "//div/h2[contains(text(), 'Logowanie')]"
+                "//..//form/div/input[contains(@id, 'id_password')]"
+            )
+        elem.send_keys("STXNext123!")
```
Comment (sabinaczopik) [inline behave/sample_test.py]: the same as above

File: behave/sample_test.py [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        elem = driver.find_element_by_xpath("//button[contains(text(), 'Zaloguj')]")
+        elem.click()
```
Comment (sabinaczopik) [inline behave/sample_test.py]: the same as above

File: behave/sample_test.py [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        self.assertIn("Volontulo", driver.title)
+        elem = WebDriverWait(driver, 10).until(
```
Comment (sabinaczopik) [inline behave/sample_test.py]: unused variable

File: behave/sample_test.py [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+        driver = self.driver
+        driver.get("http://volontuloapp.org")
+        elem = driver.find_element_by_xpath(
+                "//div/a[contains(text(), 'Zaloguj się')]")
+        elem.click()
+        elem = driver.find_element_by_xpath(
+                "//div/h2[contains(text(), 'Logowanie')]"
+                "//..//form/div/input[contains(@id, 'id_email')]"
+            )
+        elem.send_keys("volontulo.personas+user@o2.pl")
```
Comment (sabinaczopik) [inline behave/sample_test.py]: the same as above

**Comment by magul** [comment]
We're abandoning Behave infrastructure for now. I'm closing it now, will be revaluate it in unknown future.

**Pull request closed without merging**